tests/SprykerSdkTest/Zed/AppSdk/Business/AppAsyncApiFacadeTest.php

```diff
@@ -193,4 +193,28 @@ public function testAddAsyncApiMessageFromTransferObjectWithSpecifiedMessageName
         $this->tester->assertAsyncApiResponseHasNoErrors($asyncApiResponseTransfer);
         $this->tester->assertAsyncApiHasPublishMessageInChannel($asyncApiRequestTransfer->getTargetFile(), 'FooBar', $asyncApiMessageTransfer->getChannel()->getName());
     }
+
+    /**
+     * @return void
+     */
+    public function testAddAsyncApiMessagePropertyAdd(): void
```

Comment:
This is not related to your task, I guess you branched of from another branch and not from master.

Reply:
IMO this method is not used, can you double-check and remove if it's not used?